checker-qual/src/main/java/org/checkerframework/checker/mustcallonelements/qual/MustCallOnElements.java

```diff
@@ -0,0 +1,38 @@
+package org.checkerframework.checker.mustcallonelements.qual;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.DefaultFor;
+import org.checkerframework.framework.qual.DefaultQualifierInHierarchy;
+import org.checkerframework.framework.qual.SubtypeOf;
+import org.checkerframework.framework.qual.TypeUseLocation;
+
+/**
+ * An expression of type {@code @MustCall({"m1", "m2"})} may be obligated to call {@code m1()}
+ * and/or {@code m2()} before it is deallocated, but it is not obligated to call any other methods.
+ *
+ * <p>This annotation is enforced by the Object Construction Checker's {@code -AcheckMustCall} mode.
```

Comment:
The Object Construction Checker no longer exists. This should be a reference to the RLC, not to the now-defunct OCC.

Reply:
I assume that you copied this text from `@MustCall`'s docs; I opened a PR to fix those, too: https://github.com/typetools/checker-framework/pull/6687